app/views/catalog/_previous_next_doc.html.erb

```diff
@@ -1,18 +1,20 @@
 <% #Using the Bootstrap Pagination class  -%>
-<% if @previous_document || @next_document %>
-	<div id="previousNextDocument">
+<% Deprecation.warn('', 'using id="previousNextDocument" as a selector is deprecated and will be removed in Blacklight 6.0') %>
```

Comment:
I think this unconditional deprecation warning will get annoying really fast. Can we either leave a comment (or comment out the block) instead, and remember to include it in release notes when we kill the id?